src/app/onboarding/review/page.tsx

```diff
@@ -0,0 +1,203 @@
+            </Styles.SectionField>
+          </Flex>
+        </Styles.SectionBox>
+
+        {/* legal information for attorneys */}
+        {onboarding.roles.includes('ATTORNEY') && (
+          <Styles.SectionBox>
+            <Flex $justify="between" $align="center">
+              <H2>Legal Experience</H2>
+              <Link href={flowAt(3)}>
+                <Icon type="edit" />
+              </Link>
+            </Flex>
+            <Flex>
+              <Styles.SectionField>
+                <H4>Which state are you barred in?</H4>
+                <P>{onboarding.profile.state_barred || 'N/A'}</P>
+              </Styles.SectionField>
+            </Flex>
+            <Flex>
+              <Styles.SectionField>
+                <H4>What is your attorney bar number?</H4>
+                <P>{onboarding.profile.bar_number || 'N/A'}</P>
+              </Styles.SectionField>
+            </Flex>
+            <Flex>
+              <Styles.SectionField>
+                <H4>
+                  Are you registered by the Executive Office of Immigration
+                  Review?
+                </H4>
+                <P>
+                  {formatTruthy(
+                    onboarding.profile.eoir_registered,
+                    'Yes',
+                    'No',
+                    'N/A',
+                  )}
+                </P>
+              </Styles.SectionField>
+            </Flex>
+          </Styles.SectionBox>
+        )}
+
+        {/* legal experience for legal fellows */}
+        {onboarding.roles.includes('LEGAL_FELLOW') && (
+          <Styles.SectionBox>
+            <Flex $justify="between" $align="center">
+              <H2>Legal Experience</H2>
+              <Link href={flowAt(3)}>
+                <Icon type="edit" />
+              </Link>
+            </Flex>
+            <Flex>
+              <Styles.SectionField>
+                <H4>When do you expect to be barred?</H4>
+                <P>
+                  {onboarding.profile.expected_bar_date
+                    ? parseDate(new Date(onboarding.profile.expected_bar_date))
+                    : 'N/A'}
+                </P>
+              </Styles.SectionField>
+            </Flex>
+            <Flex>
+              <Styles.SectionField>
+                <H4>
+                  Are you registered by the Executive Office of Immigration
+                  Review?
+                </H4>
+                <P>
+                  {formatTruthy(
+                    onboarding.profile.eoir_registered,
+                    'Yes',
+                    'No',
+                    'N/A',
+                  )}
+                </P>
+              </Styles.SectionField>
+            </Flex>
+          </Styles.SectionBox>
+        )}
+      </Flex>
+
+      <Flex $gap="40px">
+        <BigLinkButton href={backlinkHref}>Back</BigLinkButton>
+        <BigBlueButton onClick={onSubmit}>Submit</BigBlueButton>
```

Comment:
I was able to submit without filling out the city field, but this is also because the user is currently able to navigate to onboarding pages even if the current page isn't completely filled out yet (progress bar loophole mentioned in worksesh). 

should we disable the "Submit" button if there is missing data, and add more explicit warnings about missing info on the Review page? I can add this to the agenda for Sunday's worksesh so we can discuss.